### PR TITLE
Translation and corrections of monero-core_es

### PR DESCRIPTION
**Pull request by BlackLotus64**
I've translated the missing lines of the gui file for the Spanish language, and also made a review of the already translated lines, all with Linguist QT. Everything seems to be OK but I haven't compiled the GUI since I read that there has been some problems with that.

Thanks to @lh1008 and @erciccione for the support, if you want to review the commit you're more than welcome, but I took my time checking everything so there won't be many mistakes :)

### CODE REVIEW

File: translations/monero-core_es.ts [hunk 1]
```diff
@@ -1043,37 +1043,37 @@
...
         <location filename="../pages/settings/SettingsInfo.qml" line="60"/>
         <source>GUI version: </source>
-        <translation type="unfinished"></translation>
+        <translation>Versión de GUI:</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsInfo.qml" line="89"/>
         <source>Embedded Monero version: </source>
-        <translation type="unfinished"></translation>
+        <translation>Versión embedida de Monero:</translation>
```
Comment (lh1008) [inline translations/monero-core_es.ts]: Missing space after ": "

File: translations/monero-core_es.ts [hunk 2]
```diff
@@ -1043,37 +1043,37 @@
...
     </message>
     <message>
         <location filename="../pages/settings/SettingsInfo.qml" line="118"/>
         <source>Wallet path: </source>
-        <translation type="unfinished"></translation>
+        <translation>Dirección de monedero:</translation>
```
Comment (lh1008) [inline translations/monero-core_es.ts]: Missing space after ": "

File: translations/monero-core_es.ts [hunk 3]
```diff
@@ -1043,37 +1043,37 @@
...
     </message>
     <message>
         <location filename="../pages/settings/SettingsInfo.qml" line="155"/>
         <source>Wallet creation height: </source>
-        <translation type="unfinished"></translation>
+        <translation>Altura de creación del monedero:</translation>
```
Comment (lh1008) [inline translations/monero-core_es.ts]: Missing space after ": "

File: translations/monero-core_es.ts [hunk 4]
```diff
@@ -1043,37 +1043,37 @@
...
         <location filename="../pages/settings/SettingsInfo.qml" line="165"/>
         <source> &lt;a href=&apos;#&apos;&gt; (Click to change)&lt;/a&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation> &lt;a href=&apos;#&apos;&gt; (Clic para cambiar)&lt;/a&gt;</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsInfo.qml" line="167"/>
         <source>Set a new restore height:</source>
-        <translation type="unfinished"></translation>
+        <translation>Establecer nueva altura de restaurado:</translation>
```
[the remark this answers is not in the document]
Comment (lh1008) [reply translations/monero-core_es.ts]: ok

File: translations/monero-core_es.ts [hunk 5]
```diff
@@ -1043,37 +1043,37 @@
     <message>
         <location filename="../pages/settings/SettingsInfo.qml" line="60"/>
         <source>GUI version: </source>
-        <translation type="unfinished"></translation>
+        <translation>Versión de GUI:</translation>
```
[the remark this answers is not in the document]
Comment (erciccione) [reply translations/monero-core_es.ts]: @BlackLotus64 Don't know what you mean with that, but linguist should tell you that the string is not ending in the same way of the source text. Please add the witespace here and in the other strings that @lh1008 pointed out

[the remark this answers is not in the document]
Comment (BlackLotus64) [reply translations/monero-core_es.ts]: Oh OK now that I see, yes, the missing space is in my version. I saw some lines that lacked that space and corrected them, but it seems there were some others that didn't notice 😅

File: translations/monero-core_es.ts [hunk 6]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE TS>
-<TS version="2.1" language="es_ES">
+<TS version="2.0" language="es_ES">
```
Comment (lh1008) [inline translations/monero-core_es.ts]: Not sure why it changed to "2.0".

File: translations/monero-core_es.ts [hunk 7]
```diff
@@ -1085,235 +1085,243 @@ The following information will be deleted
...
         <location filename="../pages/settings/SettingsInfo.qml" line="202"/>
         <source>Invalid restore height specified. Must be a number.</source>
-        <translation type="unfinished"></translation>
+        <translation>Altura de restaurado especificada inválida. Debe ser un número.</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsInfo.qml" line="236"/>
         <source>Wallet log path: </source>
-        <translation type="unfinished"></translation>
+        <translation>Dirección de registro del monedero:</translation>
```
Comment (lh1008) [inline translations/monero-core_es.ts]: Missing another space oups...

File: translations/monero-core_es.ts (hunk 6 above)
Comment (erciccione) [reply translations/monero-core_es.ts]: that's not a problem

Comment (lh1008) [reply translations/monero-core_es.ts]: Ok :)

File: translations/monero-core_es.ts (hunk 7 above)
Comment (BlackLotus64) [reply translations/monero-core_es.ts]: Fixing this and pushing again, thanks

Comment (BlackLotus64) [reply translations/monero-core_es.ts]: Found some other missing spaces with the Search tool looking at ':<', and changed just the ones that are missing. Left the other ones that are pair (the source text doesn't include a space) intact.

**Pull request merged**